roles/engine/files/pm2.config.cjs

```diff
@@ -3,7 +3,7 @@ module.exports = {
     {
       name: 'ota',
       script: 'npm',
-      args: 'run track',
+      args: 'run start:schedule',
```

Comment:
Why not call the `ota` CLI directly?

Reply:
As it is the declarations that declare `@opentermsarchive/engine` as a dependency, they must handle the mapping from the engine CLI to the commands expected by pm2.